packages/ListBox/stories/examples/Keyboard/index.js

```diff
@@ -0,0 +1,337 @@
+            be the Filter input. Clicking again up should result in a NO ACTION, the focus should remain on the input.
+          </CheckListItem>
+        </Block>
+      </Blocks>
+    </>
+  );
+}
+
+export function WithDividers() {
+  return (
+    <>
+      <h2>With Dividers</h2>
+      <Blocks>
+        <Block>
+          <h3>Single</h3>
+          <Card>
+            <ListBox>
+              <ListBox.Filter />
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+          <h3>Multi</h3>
+          <Card>
+            <ListBox isMulti>
+              <ListBox.Filter />
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            Divider Item should be ignore while pressing the <Up /> and <Down /> keys.
+          </CheckListItem>
+        </Block>
+      </Blocks>
+    </>
+  );
+}
+
+export function WithDisabled() {
+  return (
+    <>
+      <h2>With Disabled Items</h2>
+      <Blocks>
+        <Block>
+          <h3>Single</h3>
+
+          <Card>
+            <ListBox>
+              <ListBox.Filter />
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              <ListBox.Option isDisabled>Achilles</ListBox.Option>
+              <ListBox.Option isDisabled>Odysseus</ListBox.Option>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+          <h3>Multi</h3>
+
+          <Card>
+            <ListBox isMulti>
+              <ListBox.Filter />
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              <ListBox.Option isDisabled>Achilles</ListBox.Option>
+              <ListBox.Option isDisabled>Odysseus</ListBox.Option>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            The user is allowed to navigate over disabled elements but can not selected them or interact with them. This
+            might required further discussion of what we really want to do.
```

Comment:
This works nicely for the multiselect version, but the focus gets stuck and won't pass disabled items for the single version.